web/src/refresh-components/Modal.tsx

```diff
@@ -7,6 +7,7 @@ import { IconProps } from "@/icons";
 import Text from "@/refresh-components/texts/Text";
 import IconButton from "@/refresh-components/buttons/IconButton";
 import SvgX from "@/icons/x";
+import Truncated from "./texts/Truncated";
```

Comment:
Could you use absolute paths? Just makes the moving around of files (if ever needed) easier.